plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1473,6 +1469,12 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
         return false
       }
 
+      // Skip invalid distance with non-finite values
+      if (!distanceX.isFinite() || !distanceY.isFinite()) {
```

Comment:
assuming we had similar fix before - are we sure now that we check all the places in gestures? 😄

Reply:
No, I guess we need an audit to the gestures plugin in general, and also I've ticketed out the improvements to our code generator to guard these early.

Reply:
👍